test/integration/features/basic.feature

```diff
@@ -29,6 +29,10 @@ Feature: Basic
     Given Minishift has state "Does Not Exist"
      When executing "minishift start" succeeds
```

Comment:
```Importing openshift/origin-docker-registry:v3.6.1 . CACHE MISS``` is little hard to understand. Would you please confirm the behaviors what i  am seeing in ```m start``` log. 
```make integration GODOG_OPTS="-tags basic"```
```                        
[...]    
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 0% used OK
    Importing openshift/origin:v3.6.1 . CACHE MISS
    Importing openshift/origin-docker-registry:v3.6.1 . CACHE MISS
    Importing openshift/origin-haproxy-router:v3.6.1  CACHE MISS
-- OpenShift cluster will be configured with ...
    Version: v3.6.1
     [...]
```

Reply:
@amitkrout `CACHE MISS` means the images are not cached yet, i.e you are running `minishift start` for the first time with `image-caching` so this time it will cache the images and from the next start it will use the cached images, and you should not get this again.

Reply:
It is expected at the moment. I think we shouldn't show it on first fresh start.

Reply:
@amitkrout As per discussion with @LalatenduMohanty , let's wait for some time . If community reports back, then we can change this behavior.

Reply:
Thanks for the clarification. I was confused with ```CACHE MISS```. Can we use better phrase for this like ```SEARCHING CACHE```
@robin-owen Please suggest